__tests__/person.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../person'
+require_relative '../nameable'
+
+describe Person do
+  context 'Test the Person class' do
+    person = Person.new(50, 'Blessed Jason Mwanza', true)
+
+    it 'has class Person' do
+      expect(person.class) == Person
+    end
+
+    it 'must the right name, age and permission' do
+      expect(person.age).to eq 50
+      expect(person.name).to eq 'Blessed Jason Mwanza'
+      expect(person.parent_permission).to eq true
+    end
+  end
+end
```

Comment:
- [ ] If possible, as it is a best practice, it will be nice to have a 100% test coverage, so let's try to write tests for all the `methods` in the various classes. For this particular class, it will be nice to add tests for the following cases; 
- `When adding a new rentals (e.g. "The add_rental method returns a rental and adds it to the book")`,
- `When checking permissions (e.g. "The can_use_services? method returns false if under age and no parent permission", "The can_use_services? method returns true if over age", "The can_use_services? method returns true if has parent permission", `,
- `When validating name(e.g."The validate_name method corrects the name")`

Reply:
Thanks a lot for your professional review @Mihndim2020 . We'll go on implemented the suggested changes
>HappyCoding